org.tenacityaudio.Tenacity.yaml

```diff
@@ -0,0 +1,191 @@
+app-id: org.tenacityaudio.Tenacity
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: tenacity
+rename-icon: tenacity
+rename-desktop-file: tenacity.desktop
+rename-appdata-file: tenacity.metainfo.xml
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network # Audacity uses network sockets for IPC
+  - --socket=pulseaudio
+  - --device=all # ALSA
+  - --filesystem=host
+  # mod-script-pipe provides named pipes in /tmp
+  - --filesystem=/tmp
+  - --env=LD_LIBRARY_PATH=/app/lib/audacity
+  - --env=ALSA_CONFIG_PATH=
+  - --env=LADSPA_PATH=/app/extensions/Plugins/ladspa
+  - --env=LV2_PATH=$HOME/.lv2:/app/extensions/Plugins/lv2
+  # Audacity uses VST_PATH
+  - --env=VST_PATH=/app/extensions/Plugins/vst
+  - --env=VST3_PATH=/app/extensions/Plugins/vst3
+  # JACK support via pipewire
+  - --filesystem=xdg-run/pipewire-0
+
+add-extensions:
+  # org.audacityteam.Audacity.Codecs:
+  #   directory: lib/codecs
+  #   add-ld-path: lib
+  #   bundle: true
+  #   autodelete: true
+  org.freedesktop.LinuxAudio.Plugins:
+    directory: extensions/Plugins
+    version: '22.08'
+    add-ld-path: lib
+    merge-dirs: ladspa;lv2;vst;vst3
+    subdirectories: true
+    no-autodownload: true
+
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /share/aclocal
+  - /share/man
+  - '*.la'
+  - '*.a'
+
+modules:
+  - name: wxwidgets
+    rm-configure: true
+    config-opts:
+      - --with-libpng
+      - --with-zlib
+      - --with-cxx=17
+      - --disable-sdltest
+      - --disable-webview
+      - --disable-webviewwebkit
+      - --disable-ribbon
+      - --disable-propgrid
+      - --disable-richtext
+      - --with-expat=builtin
+      - --with-libiconv=/usr
+    cleanup:
+      - /share/bakefile
+    sources:
+      - type: git
+        url: https://github.com/wxWidgets/wxWidgets
+        tag: v3.2.2.1
+        commit: a812fffda3fe686c94e24bff27e8effd96e4de64
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
+
+  - deps/lame-3.100.json
+
+  - name: libid3tag
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://codeberg.org/tenacityteam/libid3tag.git
+        tag: 0.16.2
+        commit: 3df1882b76c2f44a274af1af33497125c95679b9
+        x-checker-data:
+          type: git
+          tag-pattern: ^([\d.]+)$
+
+  - shared-modules/libmad/libmad.json
+  - shared-modules/linux-audio/lv2.json
+  - shared-modules/linux-audio/lilv.json
+  - deps/suil.json
```

Comment:
suil implementation in Audacity is broken and the flatpak carries patches. (upstream ignored the PR)

See the second chunk of the patch

https://github.com/flathub/org.audacityteam.Audacity/blob/master/patches/audacity-suil-fix.patch

The symptom is that the UI for LV2 plugins fail to log. At one point there was output on the console for that.

Reply:
(the patch is specific to flatpak, but the fix I had submitted upstream was more generic)

Reply:
That was a PR https://github.com/audacity/audacity/pull/590

Reply:
> That was the PR https://github.com/audacity/audacity/pull/590

I highly recommend you rebase this PR against Tenacity and make a PR. These changes look reasonable to merge upstream in Tenacity. If it was already merged in tenacity-legacy, I'll remerge it.

As 1.3 beta 3 has already been tagged, I suggest the patch be (re-)added for now until we update the Flatpak again.

Reply:
Maybe you could submit an MR to [Tenacity](https://codeberg.org/tenacityteam/tenacity)? (I'm only here to create the flatpak, but I'm not familiar with the codebase)

Reply:
it's up to you if you want things to work or not.